mpa/det/semisl/trainer.py

```diff
@@ -0,0 +1,169 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+import numbers
+import os
+import os.path as osp
+import time
+import glob
+
+import torch
+import torch.distributed as dist
+import torch.multiprocessing as mp
+from mmcv.utils import get_git_hash
+from mmdet import __version__
+from mmdet.apis import train_detector
+from mmdet.datasets import build_dataset
+from mmdet.models import build_detector
+from mmdet.utils import collect_env
+# TODO[JAEGUK]: Remove import detection_tasks
```

Comment:
Can we remove now?

Reply:
No, It will be removed when mpa is merged with otx or Inhyuk's PR